src/shared/components/common/language-select.tsx

```diff
@@ -104,6 +82,40 @@ export class LanguageSelect extends Component<LanguageSelectProps, any> {
     );
   }
 
+  get selectBtn() {
+    let selectedLangs = this.props.selectedLanguageIds;
+    let filteredLangs = selectableLanguages(
+      this.props.allLanguages,
+      this.props.siteLanguages,
+      this.props.showAll,
+      this.props.showSite,
+      UserService.Instance.myUserInfo
+    );
+
+    return (
+      <select
+        className={classNames("lang-select-action", {
+          "form-control custom-select": !this.props.iconVersion,
+          "btn btn-sm text-muted": this.props.iconVersion,
```

Comment:
@dessalines Does this need `btn` classes here? It makes it look a little strange with the transparent border etc.:

<img width="187" alt="Screenshot 2023-06-04 at 4 25 58 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/b148bc9e-1d1d-4d63-bdfa-847cd8d21c97">

Reply:
Probably not, I'm open to changes if something looks better.